.github/pull_request_template.md

```diff
@@ -1,11 +1,13 @@
 [master < Epic] PR
 - [ ] Check, and update documentation if necessary
-- [ ] Update [changelog](https://docs.memgraph.com/memgraph/changelog)
 - [ ] Write E2E tests
 - [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch and the Epic branch
 - [ ] Provide the full content or a guide for the final git message
 
 [master < Task] PR
 - [ ] Check, and update documentation if necessary
-- [ ] Update [changelog](https://docs.memgraph.com/memgraph/changelog)
 - [ ] Provide the full content or a guide for the final git message
+
+
+To keep docs changelog up to date, one more thing to do:
+- [ ] Write a release note here
```

Comment:
and tag me (docs) to check

Reply:
Where to tad you? In the slack channel or here on github in comments at the end somewhere?

Reply:
I think the shortest path is to do it on github in the comments and we can continue the discussion about the note in the comments

Reply:
@vpavicic added this part